sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -239,8 +242,8 @@ CosmosAsyncContainer setId(String id) {
      * @return an {@link Flux} containing one or several feed response pages of the
      *         read cosmos items or an error.
      */
-    public Flux<FeedResponse<CosmosItemProperties>> readAllItems() {
-        return readAllItems(new FeedOptions());
+    public <T> Flux<FeedResponse<T>> readAllItems(Class<T> klass) {
```

Comment:
nit: klass -> classType?

Reply:
We are using klass everywhere else, so followed the same pattern